src/app/views/admin-views/category/category.component.ts

```diff
@@ -7,7 +9,7 @@ import { Component, OnInit } from '@angular/core';
 })
 export class CategoryComponent implements OnInit {
 
-  constructor() { }
+  constructor( private modal: NgbActiveModal) { }
```

Comment:
Remove unwanted service integration

Reply:
NgbActiveModal
 this is for cancle the pop up,i tested,its not working without ngbactiveModal.